main/src/ca/uwaterloo/flix/api/lsp/provider/HighlightProvider.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Magnus Madsen
+ * Copyright 2024 Magnus Madsen, Alexander Dybdahl Troelsen
```

Comment:
I think putting on two lines would be more readable and also preserve the dates.

Reply:
Oh, right. Yeah, I'll do that. Like A or B?
A:
```
Copyright 2020 Magnus Madsen
Copyright 2024 Alexander Dybdahl Troelsen?
```

B:
```
Copyright 2020 Magnus Madsen
          2024 Alexander Dybdahl Troelsen
```

Reply:
I think B is the easiest